cmd/veracity/main.go

```diff
@@ -32,6 +32,7 @@ func main() {
 	if envikwid == "1" || strings.ToLower(envikwid) == "true" {
 		ikwid = true
 	}
+	ikwid = true
```

Comment:
Doesn't 'this force IKWID on all of the time?

Reply:
ah yes, good spot. accidental dev hack